app/assets/javascripts/misc/page-unload-warning.js

```diff
@@ -1,5 +1 @@
-function pageUnloadWarning() {
-  return true;
-}
-
-window.onbeforeunload = pageUnloadWarning;
+window.onbeforeunload = () => true;
```

Comment:
this is cool, but doesn't seem to be related to this PR. Can you amend the commit message and/or PR description to offer some rationale for including it?